src/SolutionTemplate/5.2/uno52blank/Directory.Build.props

```diff
@@ -5,6 +5,7 @@
     <Nullable>enable</Nullable>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <UnoSdkDebugging>true</UnoSdkDebugging>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Should we add this [here](https://github.com/unoplatform/uno/blob/8d386cad726054a15f53c8803941c5c8b77caed8/src/SolutionTemplate/Directory.Build.props) instead?